src/messagebuffer.cpp

```diff
@@ -59,7 +60,7 @@ void MessageBuffer::handle_string(std::string s)
         // we reach the length of the body as provided in the Content-Length
         // header.
         auto content_length = std::stoi(m_headers["Content-Length"]);
-        if (m_raw_message.length() == content_length) {
+        if (m_raw_message.length() >= content_length) {
```

Comment:
Can you explain this change?

Reply:
Was part of debugging. Will revert this and the iostream.